lib/FFI/Platypus/Type/Enum.pm

```diff
@@ -245,9 +246,22 @@ Perl:
    'better',
    [best => 12],
  );
- 
+
  $ffi->attach( f => [ 'foo_t' ] => 'void' );
 
+=item casing
+
+[version 0.06]
```

Comment:
Does this need a space, so it indents like code?

Reply:
no.